app/models/lpp/LPPDetails.scala

```diff
@@ -45,7 +45,8 @@ case class LPPDetails(
                        principalChargeDueDate: LocalDate,
                        penaltyChargeReference: Option[String],
                        principalChargeLatestClearing: Option[LocalDate],
-                       LPPDetailsMetadata: LPPDetailsMetadata
+                       LPPDetailsMetadata: LPPDetailsMetadata,
+                       timeToPay: Option[Seq[TimeToPay]]
```

Comment:
Does this not break the 22-param case class limit?

Reply:
By my count this takes us to 24 params which means we are already beyond the limit 🤔 If it has been sone differently in the backend to get around this I will follow that